### PR TITLE
X1-AV-27

### PR DESCRIPTION
**Pull request by lisaundre**
updated sensor data

### CODE REVIEW

File: src/rocketData.py [hunk 1]
```diff
@@ -15,127 +13,137 @@ class RocketData:
...
+            self._data['lsm']['acceleration_z'] = az
+            self._data['lsm']['magentometer_x'] = mx
+            self._data['lsm']['magentometer_y'] = my
+            self._data['lsm']['magentometer_z'] = mz
+            self._data['lsm']['gyroscope_x'] = gx
+            self._data['lsm']['gyroscope_y'] = gy
+            self._data['lsm']['gyroscope_z'] = gz
+            self._data['lsm']['temperature'] = t
+        else: 
+            print("LSM accepts 10 values in this order: Acceleration (x, y, z), magentometer (x, y, z), gyroscope (x, y, z")
```
Comment (klemie) [inline src/rocketData.py:108]:
```suggestion
            print("LSM accepts 10 values in this order: Acceleration (x, y, z), magentometer (x, y, z), gyroscope (x, y, z)")
```
forgot a closing brace

File: src/dataHandler.py [hunk 2]
```diff
@@ -42,7 +42,7 @@ class SendData:
         Parm:
             sendingTo: can either be "blackbox", "ground"
             dataToFomat: Which sensor is needed to be called
-    
+   test
```
Comment (klemie) [inline src/dataHandler.py:45]: test?

**Review by klemie — CHANGES_REQUESTED** [review]
Looks good on my first look over just some small things. Will look deeper tomorrow

File: src/dataHandler.py (hunk 2 above)
Comment (krislemie) [reply src/dataHandler.py:45]: Test!

**Pull request merged**